hayeon/baekjoon/자료구조/1920-수-찾기.js

```diff
@@ -0,0 +1,35 @@
+const readline = require("readline");
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const input = [];
+
+rl.on("line", function (line) {
+  input.push(line);
+}).on("close", function () {
+  // 풀이
+  const arrayA = new Set(input[1].split(" ").map(Number));
+  const arrayM = input[3].split(" ").map(Number);
+  let result = [];
+
+  arrayM.forEach((number) => {
+    arrayA.has(number) ? result.push(1) : result.push(0);
+  });
+
+  console.log(result.join("\n"));
```

Comment:
```suggestion
  const integerSet = new Set(input[1].split(" "));
  const comparList = input[3].split(" ");
  const result = [];

  comparList.forEach((number) => {
    integerSet.has(number) ? result.push(1) : result.push(0);
  });

  console.log(result.join("\n"));
```
- 풀이 잘봤습니다. 🙇‍♂️
- 변수명이 자료구조 set인데 arrayA라는 점 목적과 의미에 맞지 않는 변수명이라 아쉽네요 좀 더 의미있게 변수명을 작성하셨으면 좋겠습니다 ~ 🙆‍♂️
- result는 재할당되는 부분이 없기때문에 const로 선언하셔야합니다 ~

Reply:
헉.. 변수명을 저렇게 지을 수 있겠군여.... 고민하다가.. 되게..그렇게 지어버린.. result 부분은 실수했네유... 감사합니다!!👍🏻